babycenter/templates/base.html

```diff
@@ -27,8 +27,9 @@
 
     <body>
       <div class="languages l-indent f-small">
+        {% get_current_language as LANGUAGE_CODE %}
         {% for language in languages %}
-          <a href="{% url 'locale_set' language.locale %}?next={{request.path}}">{{language}}</a>
+          <a href="{% url 'locale_set' language.locale %}?next={{request.path}}" class="{% if LANGUAGE_CODE == language.locale %}active{% endif %}">{{language.locale|language_name_local}}</a>
```

Comment:
Might be able to save some bytes here if you wrote:

`{% if LANGUAGE_CODE == language.locale %} class="active" {% endif %}